examples/hata_basic_queue/bot.py

```diff
@@ -91,7 +91,7 @@ async def _join(ctx):
 @bot.commands
 @checks.guild_only()
 async def join(ctx):
-    """Join the voice channel the user is on."""
+    """Join the user's voice channel and create a lavalink session."""
```

Comment:
This comment shows up on the help command (if implemented?), the user doesn't need to know about "a lavalin session has been created".
```suggestion
    """Join the voice channel you are on."""
```

Reply:
Commited, but with `in` instead of `on`.

Reply:
```suggestion
    """Joins the voice channel you are in."""
```